conda_forge_tick/migrators/version.py

```diff
@@ -361,6 +361,22 @@ def _fmt_error_message(errors, version):
     return msg
 
 
+def check_version_matches_cadence(new_version: str, version: str, bot):
```

Comment:
We should write some tests for this function. We put in an example regex someone might use with various version strings and make sure it does what we think it should do.

Reply:
These tests could then be turned into documentation examples for the bot.

Reply:
> We should write some tests for this function. We put in an example regex someone might use with various version strings and make sure it does what we think it should do.

Right, I will think of some tests for it.

Reply:
> These tests could then be turned into documentation examples for the bot.

As an example for the regex patterns that can be added to the feedstocks configs ?

Reply:
Yup.